testing/make.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e 
+
+if [ -n "$INSTALL_TAG" -a -z "$INSTALL_ROOT" ]; then
+  INSTALL_ROOT="$WORKSPACE_ROOT/root/$INSTALL_TAG"
+fi
+
+#if [ \! -f $INSTALL_ROOT/.botan.ok ] ; then
+#  rm -f Botan-1.10.13.tgz
+#  wget 'https://botan.randombit.net/releases/Botan-1.10.13.tgz'
+#  gzip -d < Botan-1.10.13.tgz | tar xf -
+#  cd Botan-1.10.13
+#  ./configure.py --prefix="$INSTALL_ROOT"
+#  make
+#  make install
+#  cd ..
+#  touch $INSTALL_ROOT/.botan.ok
+#  echo "1.10.13" > $INSTALL_ROOT/.botan.build
+#fi
+
+if [ \! -f $INSTALL_ROOT/.softhsm2.ok ] ; then
+  rm -f softhsm-2.2.0.tar.gz
+  wget 'https://dist.opendnssec.org/source/softhsm-2.2.0.tar.gz'
+  gzip -d < softhsm-2.2.0.tar.gz | tar xf -
+  cd softhsm-2.2.0
+  ./configure --prefix="$INSTALL_ROOT" \
+              --disable-non-paged-memory \
+              --with-migrate --disable-p11-kit \
+              --with-crypto-backend=openssl \
+              --disable-ecc --disable-gost
```

Comment:
Not all test machines are/were equiped with the right openssl lib afaik/earlier tested.  This is just part of the default distribution.
For my part, not part of this issue.